plugins/modules/aws_api_gateway_domain.py

```diff
@@ -0,0 +1,355 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: aws_api_gateway_domain
+short_description: Manage AWS API Gateway custom domains
+description:
+     - Allows for the management of API Gateway custom domains. To have nice domain names for your API GW Rest APIs.
+     - AWS API Gateway custom domain setups use CloudFront behind the scenenes.
+       So you will get a CloudFront distribution as a result, configured to be aliased with your domain.
```

Comment:
```suggestion
       This results in a CloudFront distribution aliased to your domain.
```

Reply:
Won't change that, as it changes the logic of the sentence.